src/content/subgames.cpp

```diff
@@ -113,6 +114,10 @@ SubgameSpec findSubgame(const std::string &id)
 	if (user != share || user_game)
 		mods_paths.insert(user + DIR_DELIM + "mods");
 
+	for (std::string mod_path : getEnvModPaths()) {
+		mods_paths.insert(mod_path);
+	}
```

Comment:
It's not important here, but you could do less copying:
```suggestion
	for (std::string &&mod_path : getEnvModPaths()) {
		mods_paths.insert(std::move(mod_path));
	}
```

Reply:
No need to micro-optimize this IMO:
```suggestion
	for (const std::string &mod_path : getEnvModPaths()) {
		mods_paths.insert(mod_path);
	}
```

Reply:
The revised patch uses const ... & in two places. I'm not familiar enough with C++ to tell if it is defined behaviour though.